src/sgg_utils/foreup_utils.py

```diff
@@ -433,4 +433,41 @@ def get_item(token, course_id, item_id):
     if r.status_code == 200 and len(content['data']) > 0:
         item.append(content['data'])
 
-    return item
\ No newline at end of file
+    return item
+
+def get_special(token, course_id, teesheet_id, special_id):
+    '''get special for a teesheet'''
+    headers = {
+        'Content-Type': 'application/json',
+        'x-authorization': f'Bearer {token}'
+    }
+    url = f'{API_URL}/courses/{course_id}/teesheets/{teesheet_id}/specials/{special_id}'
+    r = requests.get(url, headers=headers)
+    content = json.loads(r.content)
+    return content
+
+def get_specials(token, course_id, teesheet_id, limit=100, testing=False):
```

Comment:
It looks like the foreup get all specials endpoint doesn't have any pagination - i.e. it gives you everything all at once. So you can take out the while loop and anything that uses limits. It will be a simpler function that looks similar to the `get_special` function, just without a specific special id.